cnames_active.js

```diff
@@ -957,6 +957,7 @@ var cnames_active = {
   "evolutionland": "evolutionlandorg.github.io/js-sdk",
   "evolutions": "anshumanv.github.io/evolutions-webapp",
   "ewa": "lyfeyaj.github.io/ewa",
+  "examples.whatsup": "whatsup.github.io/examples",
```

Comment:
```suggestion
  "examples.whatsup": "whatsup.github.io/examples", // noCF
```
As this is a sub-sub-domain, we're not able to provide the Cloudflare proxy for it. As such, the entry will need `// noCF` at the end of it to indicate this, please